infra/api/docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: '3'
+
+services:
+  api-id_manager:
```

Comment:
use a precise name pls.

Reply:
ID_MANAGER_API

Reply:
semnatic, it's the same thing :) I've put "api" as prefix, so all api containers will be neighbours in the list.
Prefix is the functionlity of container (in this case "api") and the last part is the name of service (id_manager)